packages/cli/bin/n8n

```diff
@@ -71,6 +71,10 @@ if (!globalThis.crypto?.getRandomValues) {
 }
 
 (async () => {
+	// Collect DB entities from modules _before_ `DbConnectionOptions` is instantiated.
+	const { BaseCommand } = await import('../dist/commands/base-command.js');
+	await new BaseCommand([], { root: __dirname }).loadModules();
+
```

Comment:
We will no longer need `BaseCommand` once we move module loading to its own class, as in PR description.